DDD Workshop.ipynb

```diff
@@ -0,0 +1,398 @@
+    "\n",
+    "#### \"Useful\" Information\n",
+    "\n",
+    "When we say \"useful\", what we want is SMART. SMART is an acronym that is generally used for goal-setting, where it stands for:\n",
+    "\n",
+    "- S: Specific\n",
+    "- M: Measurable\n",
+    "- A: Attainable\n",
+    "- R: Relevant\n",
+    "- T: Timely\n",
+    "\n",
+    "<!--  --> cite https://www.mindtools.com/pages/article/smart-goals.htm\n",
+    "\n",
+    "These are the things that you want in goal setting for sure, but we also want them in information, and it's no coincidence (p > 0.05). We want information that supports the goals you have when using it. Otherwise, information is as good as trivia, which is not bad necessarily, but useless by definition. Very briefly point by point:\n",
+    "\n",
+    "- Specific: this information should be scoped to answer the question(s) you have, as information too broad risks being less applicable to the situation\n",
+    "- Measurable: an excellent metric for what to do with your data, and the subject of the next part\n",
+    "- Attainable: similar to specificity, information too specific might be too difficult to build a solution for, or not worth the development time\n",
+    "- Relevant: if your information is not about your subject, it won't be a good basis for solutions\n",
+    "- Timely: while data being too old doesn't mean that it's wrong, it makes the correlation to today weaker and harder to build for. Likewise, a projection too far in the future might not be appropriate for your current project or plans.\n",
+    "\n",
+    "\n",
+    "#### \"Strong\" Information\n",
+    "\n",
+    "While this sounds like a dataset for powerlifting competitions, it reflects the next stage in data-driven decision making that we need to address, what do we do with the data?\n",
+    "\n",
+    "Consider the following information:\n",
+    "> Men 18-25 years old in the last 6 months were more likely to purchase jeans than men aged 25-40. \n",
+    "\n",
+    "What does a company that produces jeans do with this information? Probably they would increase brand marketing to men who are 18-25, but by how much? The information makes a distinction (more likely vs less likely), not a measurement (X percent more likely). If the information was instead:\n",
+    "> Men 18-25 years old in the last 6 months were 10 times more likely to purchase jeans than men aged 25-40.\n",
+    "\n",
+    "Ad agencies would be jumping from their seats to start hiring male actors between 18 and 25! This touches on how useful information can be, and, as a bonus, is generally a good way to tell how good the question being asked in a survey was.\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Datasets\n",
+    "\n",
+    "On our journey to data-enlightenment, we do need to start somewhere. I've enjoyed sharing comics and other cherry-picked factoids for you all, but let's now practice what we preach.\n",
+    "\n",
+    "#### Covid-19 data\n",
```

Comment:
Part one does a great job of capturing the scope of data-driven decisions, but I think we could take one of the COVID-19 or census data sections and remove them.

Reply:
Good call, I'm going to comment out the census data, the observation I had about the counts can be part of a post that discusses that, if it's something we develop in the future